1.9/src/com/github/joelgodofwar/sr/ShulkerRespawner.java

```diff
@@ -142,7 +142,7 @@ public void onCreatureSpawn(CreatureSpawnEvent e){ //onEntitySpawn(EntitySpawnEv
         	if(debug){logDebug("biome=" + entity.getWorld().getEnvironment().toString());}
         	if(entity.getWorld().getEnvironment() == Environment.THE_END){
         		if(debug){logDebug("block=" + entity.getLocation().getBlock().getType().toString());}
-        		if(entity.getLocation().subtract(0, 1, 0).getBlock().getType().toString().contains("PURPUR")||entity.getLocation().getBlock().getType().toString().contains("PURPUR")){

```

Comment:
@mateohastie 

I don't think the new blocks will trigger ever. This code only fires when an enderman tries to spawn in the end. But in the end, none of those blocks are natively found in an orientation where an enderman could spawn on top of them. I think?

Reply:
true but most of the end cities are destroy or simply using other blocks to expand the platform on the en cities is more for a commodity than anything else.